tests/_bootstrap.php

```diff
@@ -16,7 +16,7 @@
 	 */
 
 	// TEC Test Support.
-	$tec_dir         = dirname( __DIR__, 2 );
+	$tec_dir         = dirname( __DIR__, 3 ) . '/the-events-calendar';
```

Comment:
You shouldn't assume we are in the TEC directory!

Reply:
@Camwyn  It is some workaround for EVA tests. Not sure we should mess with this at this point. cc @pattihis

Reply:
The test is skipped for now, so the heck with it. It should work the same.